src/components/tests/image.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { shallow } from '../enzyme'
+import Image from '../Image'
+
+describe('Image test', () => {
+  
+  it('image renders', () => {
+    shallow(<Image src="https://picsum.photos/240/392?random=286" />)
```

Comment:
No assertion?

Reply:
I need to learn more about jest and enzyme, honestly. I hope to learn about this by example = )